packages/frontend/src/utils/wallet.ts

```diff
@@ -1671,7 +1717,7 @@ export default class Wallet {
 
     async signMessage(message, accountId = this.accountId) {
         const account = await this.getAccount(accountId);
-        const signer = account.inMemorySigner || account.connection.signer;
+        const signer = account.signerIgnoringLedger || account.connection.signer;
```

Comment:
In this case, the account is a native Near `class Account` - which I don't think will ever implement that property `signerIgnoringLedger? Since we call `this.getAccount(accountId)`.

Reply:
![Screenshot 2023-08-05 at 12 56 33 AM](https://github.com/mynearwallet/my-near-wallet/assets/125173477/912f7793-373f-45cc-81e3-4ea5e9fabbe4)

Actually, account won't have "inMemorySigner" too. so no matter what is the variable name, it is undefined, what the code really means, is "use the second parameter of the OR"